app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,6 @@
 <resources>
     <string name="device_admin_label">onebillion</string>
     <string name="device_admin_description">Admin Device Policies Description</string>
-    <string name="content_extraction" translatable="false">यह प्रक्रिया केवल एक बार की जाएगी।</string>
+    <string name="content_extraction" translatable="false">यह प्रक्रिया केवल एक बार की जाएगी। इसमें 10 मिनट लग सक्ते हैं।</string>
```

Comment:
If we are adding hindi full stop, please keep it consistent

Reply:
@ankitk20 @Shailin @harry6396 
Note this point